tests/manage/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -0,0 +1,179 @@
+"""
+A test case to verify after deleting pvc whether
+size is returned to backend pool
+"""
+import logging
+import time
+import pytest
+
+from ocs import constants, defaults
+import ocs.exceptions as ex
+from tests import helpers
+from ocsci.testlib import tier1, ManageTest
+from utility import templating
+from resources import pod
+from ocs import ocp
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_rbd_secret
+)
+
+logger = logging.getLogger(__name__)
+_templating = templating.Templating()
+
+PV = ocp.OCP(
+    kind='PersistentVolume', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+
+def check_ceph_used_space():
+    """
+    Check for the used space in cluster
+    """
+
+    # ToDo: after running IO's or deleting pod & pvc takes some time
+    time.sleep(20)
+    ct_pod = pod.get_ceph_tools_pod()
+    pods = ct_pod.exec_ceph_cmd(ceph_cmd="ceph status")
+    assert pods is not None
+    used = pods['pgmap']['bytes_used']
+    GB = (1024 * 1024 * 1024)
```

Comment:
please move to constant.py in the form of:
GB = 1024 ** 3

Reply:
done